### ISSUE TITLE
Script resilience to version updates

### ISSUE THREAD

**Issue by alix-dumoulin** [sagepublishing/text_cleaning#3]
**TLDR**: some scripts found on GitHub / StackOverflow use deprecated methods/functions/libraries 

**Most common cases so far**:
* Mismatch between script and version installed using pip, can lead to methods not existing, old set of parameters because the function changed, requirements for the package not being installed 
* Script using libraries that are not maintained anymore
* Scripts importing a module that is not part of the package anymore (or may have a new name?)

**Example**:
[This script](https://gist.github.com/ivandeex/f903347745688da9bd5d) which converts pdf to html/XML 
* imports `process_pdf()` which after [some research](https://stackoverflow.com/questions/57079815/how-to-fix-cannot-import-name-process-pdf-from-pdfminer-pdfinterp-error) turns out to have been replaced with `PDFPage.get_pages()`
* and has a parameter codec for encoding which is not part of the set of parameters in the new version
* imports a module `ply`which is not in the requirements.txt (therefore not installed via pip)

**Why it's a problem**
* Now: makes it a bit painful to get scripts running
* Later: if we integrate scripts into a product, we need to make sure they are resilient to updates of the packages they use (which is very high-maintenance)
